plugins/outputs/prometheus_client/prometheus_client_tls_test.go

```diff
@@ -0,0 +1,129 @@
+package prometheus_client_test
+
+import (
+	"crypto/tls"
+	"fmt"
+	"github.com/influxdata/telegraf/plugins/outputs/prometheus_client"
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/influxdata/toml"
+	. "github.com/onsi/gomega"
```

Comment:
We will need to remove this dependency, I don't want to add it only for testing this one aspect.

Reply:
Replaced with require